docs/recipes-annotations.rst

```diff
@@ -2,6 +2,15 @@
 
 .. _RecipesAnnotations:
 
+.. |toggleStart| raw:: html
```

Comment:
This is fine, but you haven't wrapped any of the later sample code with `|toggleStart|` & `|toggleEnd|` so it doesn't do anything. Also we may as well move the definition for this into just one place on `header.rst` & remove from `recipes-ocr.rst`, `recipes-multiprocessing.rst`, `recipes-stories.rst` ( I believe those are the only other files which use this component )

Reply:
You're quite right about this being unused; i've removed it in my tree.

But we shouldn't put this in `header.rst` or similar, because it is specific to the recipes docs - it contains `<summary><a>See recipe</a></summary>`. I was unable to find a way to specify the text at the place where `toggleStart/End` is used.